### ISSUE TITLE
More color options

### ISSUE THREAD

**Issue by AbcAeffchen** [goat1000/SVGGraph#36]
Hi,

I stumbled over two limitations for color options.

1. The median of a BoxAndWhiskerGraph has the same color as the border of the box. This is a problem if the median is on the upper or lower border of the box. There should be an option for color it separately
2. If you have multiple guideline one can name them but not color them separately.

I'm not sure if there are more functions that could use an additional color option.

As always, let me know if I should try to code something for this.

Best,
Alex

**Comment by goat1000** [comment]
You're right about the median line colour, so I can fix that quite easily.

Guidelines can be coloured separately, so I probably haven't documented them very well. `guideline_colour` will set the default colour of guidelines, and a `colour` entry in a guideline's array will set the colour just for that line and its text:

````PHP
$options = [
  'guideline' => [
    [ 120, 'Line at 120'],
    [ 100, 'Line at 100', 'colour' => 'red' ],
  ],
  'guideline_colour' => 'blue',
];
````

**Comment by goat1000** [comment]
I've added `median_colour` and `median_dash` options in 038d0ad.

I haven't used this type of graph for anything serious, so I don't know if being able to set these individually using structured data would be useful. If you can see a use for it, I'll add support for that too.
